modules/integration/src/test/resources/log4j.properties

```diff
@@ -36,3 +36,5 @@ log4j.logger.org.apache.zookeeper.ZooKeeper=WARN
 log4j.logger.org.apache.curator.framework.recipes.cache.PathChildrenCache=FATAL
 log4j.logger.org.apache.fluo=ERROR
 
+
```

Comment:
Why is this random space here?

Reply:
Oh, I modified that file while debugging this change.  I can remove those.